### PR TITLE
checkActivities Helper Function

### PR DESCRIPTION
**Pull request by matthewrsj**

### CODE REVIEW

**Comment by matthewrsj** [comment]
@tschuy @Kennric @iankronquist

File: tests/helpers.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    describe('checkActivities helper function', function() {
+        it('Returns true when passed valid activity slugs', function(done) {
+            expect(helper.checkActivities(['developing',
+                                           'meeting',
+                                           'planning'])).to.equal(true);
+            done();
+        });
+
+        it('Returns false when passed invalid activity slug', function(done) {
+            expect(helper.checkActivities('developing',
```
Comment (tschuy) [inline tests/helpers.js]: This isn't a list.

File: tests/helpers.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+                                           'meeting',
+                                           'notanactivity')).to.equal(false);
+            done();
+        });
+
+        it('Returns false when passed null parameter', function(done) {
+            expect(helper.checkActivities('developing',
```
Comment (tschuy) [inline tests/helpers.js]: also not a list.

**Comment by tschuy** [comment]
As discussed in meatspace, this function will need to be asynchronous since it touches the database, and so the tests and the function need to be rewritten.

**Comment by tschuy** [comment]
Closed because we need a new branch for making this async.

**Pull request closed without merging**